.github/CODEOWNERS.md

```diff
@@ -0,0 +1,75 @@
+# Code Owners
+# Code owners have the ability to approve a pull request that has made changes in the below files or package(s).
```

Comment:
Generally, I try to avoid repeating the title as the first words of the first sentences. It's a nit, but, it reads awkward to me.  Suggest rewording.

Reply:
Removed the title in favor of the paragraph